docs/using-connector.md

```diff
@@ -445,9 +445,21 @@ val df = sqlContext.read
 In the previous example, the query times, `CAST('$from' AS TIMESTAMP)` and `CAST('$to' AS TIMESTAMP)`, are Timestamps which are cast from a Long integer since the datetime values in `df` are stored as Timestamps.
 
 
-## TS Table Range Query Partitioning
+## Partitioning for Riak TS Table Queries
 
-Riak TS range queries are limited to a maximum of 5 quanta (see http://docs.basho.com/riakts/latest/using/querying/). To work around this limitation or simply achieve higher read performance, large ranges can be split into smaller sub-ranges at partitioning time.
+For native Riak TS range queries (without Dataframes) single partition will be created. For Dataframe reads there are two options.
+
+### Coverage Plan Based Partitioning
+
+By default all Dataframe reads will use coverage plan based partitioner. 
+It will split initial range query into subranges each containing period of 1 quantum. All the data from the subrange is stored on the same Riak node. Subranges are then grouped by host and split into number of partitons(spark.riak.input.split.count) in the way that each partition queries single Riak node.
```

Comment:
split into 'a' number of partitions...
maybe change 'in the way' to 'such that'